.github/workflows/node.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
         run: |
           flatpak --user remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
           flatpak --user install -y flathub \
-            org.freedesktop.{Platform,Sdk{,.Extension.node{14,16}}}//21.08
+            org.freedesktop.{Platform,Sdk{,.Extension.node{14,16}}}//22.08
```

Comment:
This should also update `tools/lockfile-utils.sh` (as well as adding node18 support).

Reply:
Ugh, I've forgot about the script because I didn't use it. Done.